includes/polling/os/luminato.inc.php

```diff
@@ -0,0 +1,4 @@
+<?php
+$version = preg_replace('/[\r\n\"]+/', ' ', snmp_get($device, "swVersion.0", "-OQv", "TELESTE-LUMINATO-MIB"));
```

Comment:
Can you convert all of these snmp_gets into snmp_multi_get_oid please. Saves on SNMP queries.

Also, don't prepend Teleste to the hardware, it's supposed to just describe the model / hardware.